order-book/src/main/java/com/market/banica/order/book/model/ItemMarket.java

```diff
@@ -37,22 +54,116 @@ public void removeUntrackedItem(String itemName) {
         allItems.remove(itemName);
     }
 
+    public void updateItem(Aurora.AuroraResponse response) {
+        if (response.getMessage().is(TickResponse.class)) {
+            TickResponse tickResponse;
+
+            try {
+                tickResponse = response.getMessage().unpack(TickResponse.class);
+            } catch (InvalidProtocolBufferException e) {
+                throw new IncorrectResponseException("Response is not correct!");
+            }
+
+            Item item = new Item();
```

Comment:
this item population could be extracted in a separate metode

Reply:
Method extracted.